public/app/features/alerting/unified/utils/cloud-alertmanager-notifier-types.ts

```diff
@@ -320,7 +320,11 @@ export const cloudNotifierTypes: Array<NotifierDTO<CloudNotifierType>> = [
         'max_alerts',
         'Max alerts',
         'The maximum number of alerts to include in a single webhook message. Alerts above this threshold are truncated. When leaving this at its default value of 0, all alerts are included.',
-        { placeholder: '0', validationRule: '(^\\d+$|^$)' }
+        {
+          placeholder: '0',
+          validationRule: '(^\\d+$|^$)',
+          setValueAs: (value) => (typeof value === 'string' ? parseInt(value, 10) : 0),
```

Comment:
Assuming `0` means no max alerts (ie. all alerts)?

Reply:
According to the description above it does 😄 
Actually the `validationRule` pattern makes it difficult to insert anything other than a number

Reply:
According to the docs 0 = all alerts
https://prometheus.io/docs/alerting/latest/configuration/#webhook_config